src/main/java/sda/training/angry_nerds_game/game/Shot.java

```diff
@@ -0,0 +1,78 @@
+package sda.training.angry_nerds_game.game;
+
+import java.util.HashSet;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+import java.util.Set;
+
+public class Shot {
+
+    private Set<Point> shot = new HashSet<>();
+
+    int shotX;
+
+    public int getShotX() {
+        return shotX;
+    }
+
+    public int getShotY() {
+        return shotY;
+    }
+
+    int shotY;
+
+    public Set<Point> getShot() {
+        return shot;
+    }
+
+
+    public void genarateShotLine(Integer velocityX, Integer velocityY, Integer wind, Integer gravity) {
+
+        int maxX = SingletonGameConfig.getInstance().getMaxX();
+        int posX = 0;
+        int posY = 0;
+
+        while(posX < maxX || posY > 0) {
+            posX = posX + velocityX;
+            posY = posY + velocityY;
+            shot.add(new Point(posX, posY, '*'));
+            velocityX = velocityX + wind;
+            velocityY = velocityY - gravity;
+            if (velocityX < 0) {
+                velocityX = 0;
+            }
+        }
+    }
+
+    public void enterShotValues() {
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Podaj Paramerty Strzału X: ");
+        while (true) {
```

Comment:
staraj sie omijac while(true) i lamac to poprzez break, ja wiem ze to troche prosciej niz wprowadzic booleana ale to raczej zla praktyka, kazda petla powinna miec jakis warunek :)